puppet/zulip/manifests/postgres_appdb_tuned.pp

```diff
@@ -11,6 +11,10 @@
   'debian' => "pg_ctlcluster ${zulip::base::postgres_version} main restart",
   'redhat' => "systemctl restart postgresql-${zulip::base::postgres_version}",
 }
+$postgres_locale = $::osfamily ? {
+  'debian' => "C.UTF-8",
+  'redhat' => "en_US.UTF-8",
+}
```

Comment:
It'd be great if you can post an example of the default postgres config file for RHEL, just so I can confirm that's the default there.